spring-cloud-stream-module-launcher/README.md

```diff
@@ -69,6 +98,8 @@ $ ltc create redis redis -r
 3: Run the modules as long-running processes (LRPs) on Lattice:
 
 ````
-$ ltc create time springcloud/stream-module-launcher -e MODULES=org.springframework.cloud.stream.module:time-source:1.0.0.BUILD-SNAPSHOT -e SPRING_PROFILES_ACTIVE=cloud
-$ ltc create log springcloud/stream-module-launcher -p 8081 -e MODULES=org.springframework.cloud.stream.module:log-sink:1.0.0.BUILD-SNAPSHOT -e SPRING_PROFILES_ACTIVE=cloud
+$ ltc create time springcloud/stream-module-launcher -e SPRING_PROFILES_ACTIVE=cloud -m 1024 \
```

Comment:
Right, the explicit profile isn't expected here. But it looks like we have a bug where the `cloud` profile isn't activated by default when running on Lattice. Here is the [fix] (https://github.com/spring-cloud/spring-cloud-stream-modules/pull/25) that adds `lattice-core` dependency that activates `cloud` and `lattice` profiles when running on Lattice.